src/gwt/src/org/rstudio/studio/client/workbench/views/jobs/view/JobsDisplayImpl.java

```diff
@@ -111,7 +115,8 @@ public void showJobOutput(String id, JsArray<JobOutput> output, boolean animate)
                output.get(i).type(),
                output.get(i).output()), false /* scroll */);
       }
-      
+
+      widgets_.getOutputPanel().output_.compileCompleted();
```

Comment:
This doesn't feel right to me? `showJobOutput()` being called doesn't imply that compilation has finished.

Reply:
Show we have a separate method like `widgets_.showBufferedOutput()` or something? Which would then be called in `compileCompleted()`?

Reply:
Right, that would make more sense. It's not really done but it needs to ensure anything buffered is displayed. I'll fix this up.